shell/config/product/explorer.js

```diff
@@ -192,7 +192,30 @@ export function init(store) {
   headers(WORKLOAD_TYPES.DAEMON_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', 'Current', 'Desired', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
   headers(WORKLOAD_TYPES.REPLICA_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', 'Current', 'Desired', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
   headers(WORKLOAD_TYPES.STATEFUL_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
-  headers(WORKLOAD_TYPES.JOB, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Completions', 'Duration', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
+  headers(WORKLOAD_TYPES.JOB, [
+    STATE,
+    NAME_COL,
+    NAMESPACE_COL,
+    WORKLOAD_IMAGES,
+    WORKLOAD_ENDPOINTS,
+    'Completions',
+    {
+      name:      'duration',
+      labelKey:  'tableHeaders.duration',
+      value:     (row) => {
```

Comment:
It think we should here `getValue` instead of `value` prop. `getValue` generally represents a method while `value` represents a row property

Reply:
@aalves08  Just pushed, thanks!